app/assets/javascripts/main.coffee

```diff
@@ -21,3 +21,13 @@ $ ->
       if $searchQuery.trim().length <= 1 then false else true
 
   searchForm()
+
+  highlightAnswer = ->
+    $('.js-answer-permalink').on 'click', ->
+      $(@).parents('.answer').highlight()
+
+    anchor = location.hash
+    if anchor and anchor.match /#answer-\d/
+      $(anchor).highlight()
```

Comment:
This will highlight answer when the page load and has anchor text with it. I'm not pretty it will affect other pages if they have anchor answer id there as well. This is like page specific javascript load. I usually leave it in view template with content_for :javascript to insert into layout's head. What do you think about it? /cc @NARKOZ